### DISCUSSION TITLE
ZFS send doesn't work with unencrypted child nodes

### DISCUSSION THREAD

**Discussion by onionjake** [openzfs/zfs#12933] [General]
I think I am running into this issue on Ubuntu 21.04:

> When an encrypted ZFS dataset has unencrypted child nodes, this breaks the handling of a zfs send stream which will erroneously terminate with "inherited key must be loaded" when encountering the unencrypted dataset.

See more here:

https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=259231

I've searched through the issues on github and can't seem to find anywhere it might be patched, is this something I should file an issue about?

**Comment by AttilaFueloep** [comment]
Yes, this isn't patched and someone else opened #13033 for this.
